### DISCUSSION TITLE
real time data import from HIVE and postgresql to Duckdb

### DISCUSSION THREAD

**Discussion by aviyadav** [duckdb/duckdb#9199] [Q&A]
Can i configure real time data import from HIVE to Duckdb, also from postgresql? Use duckdb as a in-memory data aggregation solution?

**Comment by szarnyasg** [comment]
There is a PostgreSQL scanner extension available: https://duckdb.org/docs/extensions/postgres_scanner
DuckDB can read Hive-partitioned files: https://duckdb.org/docs/data/partitioning/hive_partitioning.html (this alone will not give you "real-time" import though)
